crane/cli.go

```diff
@@ -216,11 +216,11 @@ func commandAction(targetFlag string, wrapped func(unitOfWork *UnitOfWork), migh
 	wrapped(unitOfWork)
 }
 
-func excludedContainers(flag string) []string {
-	if len(flag) > 0 {
-		return cfg.ContainersForReference(flag)
+func excludedContainers(excludedReference []string) (containers []string) {
+	for _, reference := range excludedReference {
+		containers = append(containers, cfg.ContainersForReference(reference)...)
 	}
-	return []string{}
+	return containers
```

Comment:
Should be. You can use the `includes` function to have only unique containers in here.